docs/docs/alephium/handling_alph.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Handling ALPH
+
+slug: /alephium/handling_alph
+---
+
+## What is ALPH?
+
+ALPH is the native token of the Alephium blockchain. Users do not need to wrap ALPH to access their assets. This allows for direct and simplified management of ALPH tokens within the ecosystem.
```

Comment:
"This allows for direct and simplified management of ALPH tokens within the ecosystem." -> "This makes handling ALPH within the ecosystem really simple."

I don't think we should highlight what it allows and not since that decision doesn't come from us at all